### PR TITLE
Automate Dunner Release as binary, tar, deb, rpm, snap for all os/arch

### PR DESCRIPTION
**Pull request by apoorvam**
Have tested this locally and works fine, generates all packages and installers.

ToDo:
* Verify `GITHUB_TOKEN` key is added to the environment. This token is used to create releases in github repository via Travis CI. 
* Setup snap login on CI:  `snapcraft export-login snap.login` and `travis encrypt-file snap.login --add` to add the key to the travis environment. More info [here](https://goreleaser.com/ci/).

* Try out in next release

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -46,4 +48,5 @@ test-coverage:
 	$(foreach pkg, $(ALL_PACKAGES),\
 	go test -coverprofile=coverage.out -covermode=count $(pkg);\
 	tail -n +2 coverage.out >> coverage-all.out;)
-	@go tool cover -html=coverage-all.out -o coverage.html
\ No newline at end of file
+	@go tool cover -html=coverage-all.out -o coverage.html
+
```
Comment (agentmilindu) [inline Makefile]: Shall we add a command to make a release using GoReleaser?

Comment (apoorvam) [reply Makefile]: As of now, everytime we push a version tag to master, it automatically makes a release. What you think?

Comment (agentmilindu) [reply Makefile]: Yeah, how about a command `make release v.3.0.0` which create a Git tag with an appropriate tag message?

Comment (apoorvam) [reply Makefile]: `make release VERSION=v3.0.0` will add a git tag and can trigger the dunner release.

Comment (agentmilindu) [reply Makefile]: Aweomse, now this looks complete! Good job!

**Comment by agentmilindu** [comment]
Fix #8

**Pull request merged**